kpatch-build/kpatch-build

```diff
@@ -165,7 +165,7 @@ find_parent_obj() {
 }
 
 find_kobj() {
-	arg=$1
+	arg=$(echo $1 |sed 's,^\S*\.\./,,g')
```

Comment:
Hm, I think I'm missing something.  How can the argument to find_kobj ever be something like `arch/x86/kvm/../../../virt/kvm/kvm_main.o`?  The caller of find_kobj is:
```
FILES="$(find * -type f)"
cd "$TEMPDIR"
mkdir output
declare -A objnames
CHANGED=0
ERROR=0
for i in $FILES; do
	mkdir -p "output/$(dirname $i)"
	cd "$OBJDIR"
	find_kobj $i
```
It gets the filename from the "find" command.  So I would expect the argument to find_kobj to be `virt/kvm/kvm_main.o` already.